modules/claims_api/spec/factories/auto_established_claims.rb

```diff
@@ -241,7 +241,7 @@
     phase_chngd_dt { Faker::Time.backward(days: 5, period: :morning) }
     phase_type { 'Pending Decision Approval' }
     phase_type_change_ind { '76' }
-    ptcpnt_vet_id { Faker::Number.number(digits: 9) }
+    ptcpnt_vet_id { Faker::Number.number(digits: 9).to_s }
```

Comment:
This was just a mistake on my part way back.  This should be a string.  The test still returned green, but not for the reason it should have.